demo/src/edit_inline.jsx

```diff
@@ -1,56 +1,67 @@
-var React = require('react');
-var createReactClass = require('create-react-class');
-var ReactDOM = require('react-dom');
+import React from 'react';
+import ReactDOM from 'react-dom';
 
 // Note that Froala Editor has to be required separately
-require("froala-editor/js/froala_editor.min.js");
-require("froala-editor/css/froala_editor.min.css");
+import 'froala-editor/js/froala_editor.min.js';
+import 'froala-editor/css/froala_editor.min.css';
 
 // Require Font Awesome.
-require('font-awesome/css/font-awesome.css');
+import 'font-awesome/css/font-awesome.css';
 
-var FroalaEditor = require('react-froala-wysiwyg');
+import FroalaEditor from 'react-froala-wysiwyg';
 
 // Render Froala Editor component.
-var EditorComponent = createReactClass({
-  config: {
-    placeholderText: 'Add a Title',
-    charCounterCount: false,
-    toolbarInline: true,
-    events: {
-      'froalaEditor.initialized': function() {
-        console.log('initialized');
+class EditorComponent extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.config = {
+      placeholderText: 'Add a Title',
+      charCounterCount: false,
+      toolbarInline: true,
+      events: {
+        'froalaEditor.initialized': function() {
+          console.log('initialized');
+        }
```

Comment:
Best practice is to add a trailing comma (in objects), so you end up with cleaner diffs when refactoring